compass/landice/tests/humboldt/mesh.py

```diff
@@ -0,0 +1,347 @@
+        print('culling mesh')
+        dsMesh = cull(dsMesh, logger=logger)
+        print('converting to MPAS mesh')
+        dsMesh = convert(dsMesh, logger=logger)
+        print('writing grid_converted.nc')
+        write_netcdf(dsMesh, 'grid_converted.nc')
+        # If no number of levels specified in config file, use 10
+        try:
+            levels = section.get('levels')
+        except NameError:
+            levels = '10'
+        print('calling create_landice_grid_from_generic_MPAS_grid.py')
+        args = ['create_landice_grid_from_generic_MPAS_grid.py',
+                '-i', 'grid_converted.nc',
+                '-o', 'gis_1km_preCull.nc',
+                '-l', levels, '-v', 'glimmer']
+        check_call(args, logger=logger)
+
+        print('calling interpolate_to_mpasli_grid.py')
+        args = ['interpolate_to_mpasli_grid.py', '-s',
+                'humboldt_1km_2020_04_20.epsg3413.icesheetonly.nc', '-d',
+                'gis_1km_preCull.nc', '-m', 'b', '-t']
+
+        check_call(args, logger=logger)
+
+        print('calling define_cullMask.py')
+        args = ['define_cullMask.py', '-f',
+                'gis_1km_preCull.nc', '-m'
+                'distance', '-d', '50.0']
+
+        check_call(args, logger=logger)
+
+        print('calling set_lat_lon_fields_in_planar_grid.py')
+        args = ['set_lat_lon_fields_in_planar_grid.py', '-f',
+                'gis_1km_preCull.nc', '-p', 'gis-gimp']
+
+        check_call(args, logger=logger)
+
+        print('calling MpasMaskCreator.x')
+        args = ['MpasMaskCreator.x', 'gis_1km_preCull.nc',
+                'humboldt_mask.nc', '-f', 'Humboldt.geojson']
+
+        check_call(args, logger=logger)
+
+        print('culling to geojson file')
+        dsMesh = xarray.open_dataset('gis_1km_preCull.nc')
+        humboldtMask = xarray.open_dataset('humboldt_mask.nc')
+        dsMesh = cull(dsMesh, dsInverse=humboldtMask, logger=logger)
+        write_netcdf(dsMesh, 'humboldt_culled.nc')
+
+        print('Marking horns for culling')
+        args = ['mark_horns_for_culling.py', '-f', 'humboldt_culled.nc']
+        check_call(args, logger=logger)
+
+        print('culling and converting')
+        dsMesh = xarray.open_dataset('humboldt_culled.nc')
+        dsMesh = cull(dsMesh, logger=logger)
+        dsMesh = convert(dsMesh, logger=logger)
+        write_netcdf(dsMesh, 'humboldt_dehorned.nc')
+
+        print('calling create_landice_grid_from_generic_MPAS_grid.py')
+        args = ['create_landice_grid_from_generic_MPAS_grid.py', '-i',
+                'humboldt_dehorned.nc', '-o',
+                'Humboldt_1to10km.nc', '-l', levels, '-v', 'glimmer',
+                '--beta', '--thermal', '--obs', '--diri']
+
+        check_call(args, logger=logger)
+
+        print('calling interpolate_to_mpasli_grid.py')
+        args = ['interpolate_to_mpasli_grid.py', '-s',
+                'humboldt_1km_2020_04_20.epsg3413.icesheetonly.nc',
```

Comment:
Is this file just a subset of the whole GIS 1km dataset?  Did you create it separately just to allow the interpolation step to run faster?

Reply:
Yes, I just cut it out of the whole GIS 1km dataset using ncks. I could instead replace it with the whole dataset and subsample it in this script. What would you prefer?

Reply:
As long as you already have the subset created, let's just stick with it.  But add some comments here that the whole ice sheet data file could be used instead and the subset was created just to make the test run faster.

Reply:
Okay I added this comment in commit [2e78220](https://github.com/MPAS-Dev/compass/pull/272/commits/2e782205d1300d4e4f436f7c8c01cdf9852aa080).